assets/wizards/engagement/views/reader-activation/index.js

```diff
@@ -27,6 +27,7 @@ export default withWizardScreen( () => {
 	const [ config, setConfig ] = useState( {} );
 	const [ error, setError ] = useState( false );
 	const [ isActiveCampaign, setIsActiveCampaign ] = useState( false );
+	const [ hasPlugins, setHasPlugins ] = useState( false );
```

Comment:
Since `hasPlugins` is `false` by default, the UI starts in missing-plugins state and then transitions to the intended initial state only after the request completes. The default value should rather be `null`/`undefined`. 

```suggestion
	const [ hasPlugins, setHasPlugins ] = useState( null );
```